pkg/server/wire.go

```diff
@@ -383,6 +384,8 @@ var wireBasicSet = wire.NewSet(
 	userimpl.ProvideVerifier,
 	connectors.ProvideOrgRoleMapper,
 	wire.Bind(new(user.Verifier), new(*userimpl.Verifier)),
+	baseplugincontext.ProvideService,
```

Comment:
Nit. Should go into https://github.com/grafana/grafana/blob/08b18113d2a9b786e823c850faa5d23da04175a9/pkg/services/pluginsintegration/pluginsintegration.go#L58